### DISCUSSION TITLE
DB Export for Migrating From Prisma1 Cloud to Prisma 2

### DISCUSSION THREAD

**Discussion by alaindwight** [prisma/prisma#13542] [Q&A] [answered]
For migrating to from Prisma 1 Cloud, is there any way to get a Prisma 1 export into a mysql database without first spinning up a self hosted Prisma 1 server, importing there, then upgrading that to Prisma 2?

If I go in Prisma Cloud it gives me the responses from my queries in json format (1 for "users" and 1 for "posts"), is translating these into SQL then making each one a table the best bet? Or is there any other way I can take my Prisma 1 export in NDF/JSON format and convert it to SQL or something?

**Comment by nurul3101** [comment]
Hey @alaindwight 👋 
I am not aware of any way to convert NDF/JSON format to SQL at the moment.
Let us know if you found a solution to easily convert it

**Reply by alaindwight** [reply]
Thanks for the feedback. Yep, this seems to be the case as far as I know.

**Comment by andrewicarlson** [comment, marked as answer]
>For migrating to from Prisma 1 Cloud, is there any way to get prisma 1 export into a mysql database without first spinning up a self hosted prisma 1 server, importing there, then upgrading that to prisma2?

Not through the Admin panel. Send us an email at prisma1cloud@prisma.io from the email address you have on file and we can get you a mysql dump!

**Reply by alaindwight** [reply]
Thanks for clarifying. For now, since my DB is pretty small, I'm going to continue with redoing it manually. If I do run into any issues I'll definitely reach out for the sql dump!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
